README.md

```diff
@@ -1,155 +1,147 @@
+Records with a negative integer in the **amount** column would represent the number of certain fruits that were eaten at an office, while the ones with the positive amounts would represent the purchase of those fruits at the office.
 
-Having said that, you do have the full freedom to organize how many endpoints you need and however you want to organize them, as long as the frontend requirements are satisfied.
+Example:
 
-You are free to use any out of the box server or any libraries you want; we'd suggest `express` or `koa`.
+| fruit_id | location_id | amount| time                          |
+|----------|-------------|-------|-------------------------------|
+| 3        | 2           | 4     | 2018-06-09 20:00:03.629 +0200 |
+| 3        | 1           |-2     | 2017-05-09 23:33:15.233 +0200 |
 
-You DO NOT need to provide endpoints to get a list of office locations or available fruits. If you don't have the time or energy feel free to hardcode these lists, this will not be taken into consideration. The way you do hardcode them, will.
+Where the first row represents the replenishing of apples (fruit #3), at the Berlin office by purchasing 4 pieces.
 
-### 3. A nice way to run your solution
+And the second row would represent an eating of 2 pieces of apples at the Amsterdam office.
 
-Please, **provide a concise one or two line command to start your project**.
-If any explanation is necessary please create a HOWTO.md and explain how to run your solution.
+> ℹ️ Parts of the database records are generated randomly, don't expect consistancy if you recreate the container volumes.
 
-### 4. Answers to technical questions
+## 4. Constraints
 
-Please answer the following questions in a markdown file called **"ANSWERS.md"**.
+1. Build the app using **TypeScript**
+2. Preferably use **React** for the frontend
+3. The API should be a standalone server (not something like Next.js)
+4. Feel free to use **any publicly available libraries** that you are comfortable with.
 
-1. How long did you spend on the coding test?
+## 5. Delivery
 
-2. What would you add to your solution if you had more time? If you didn't spend much time on the coding test then use this as an opportunity to explain what you would add.
+When you are finished, add your project to **Github** and invite `lepaya-code-reviewer` as a **collaborator**.
 
-3. Describe your solution in plain english. Point out your design decisions and the reasoning behind them.
+> Make sure there is a consise one or two-line command to start your project. If any explanation is necessary, please create a `HOWTO.md` file and explain how to run your solution.
 
-4. Have you learned something new doing this test? If yes, describe it here.
+Email to your Lepaya HR contact informing them that you have completed your assessment.
 
-5. Describe yourself using JSON.
+## 6. Evaluation
+
+We will evaluate your solution based on the following criteria.
 
-## Instructions
+### 3.1. Real world project
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** noice!‏